src/Jobs/BasePackageJob.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Flagrow\Bazaar\Jobs;
+
+use Carbon\Carbon;
+use Flagrow\Bazaar\Composer\ComposerCommand;
+use Flagrow\Bazaar\Composer\ComposerEnvironment;
+use Flagrow\Bazaar\Composer\ComposerOutput;
+use Flagrow\Bazaar\Task;
+//use Illuminate\Bus\Queueable;
+//use Illuminate\Queue\SerializesModels;
+//use Illuminate\Queue\InteractsWithQueue;
+//use Illuminate\Contracts\Queue\ShouldQueue;
```

Comment:
Queues is not yet implemented in Flarum. So calling on it would simply do nothing or trigger an error?